app/presenters/publishing_api/topical_event_presenter.rb

```diff
@@ -41,7 +41,7 @@ def details
         details[:image] = image if item.logo_url
         details[:start_date] = item.start_date.rfc3339 if item.start_date
         details[:end_date] = item.end_date.rfc3339 if item.end_date
-        details[:ordered_featured_documents] = ordered_featured_documents
+        details[:ordered_featured_documents] = ordered_featured_documents.limit(5)
```

Comment:
Could we use the [`FeaturedLink::DEFAULT_SET_SIZE`](ihttps://github.com/alphagov/whitehall/blob/f9967fbc91b32ae6765b6ad234f5aad55c619222/app/models/organisation.rb#L504) constant for this? it'll make it less arbitrary.

Reply:
No problem 😄